### PR TITLE
Automatically calculate PC health

### PR DESCRIPTION
**Pull request by JFranklin3000**
##### In this PR
-  created prepareHealth in actor.mjs
- created steps to calculate health based on origin, condtioning and bonuses. 

##### Testing
- Validate health is being calculated correctly 
- Validate NPC are not effected.

### CODE REVIEW

File: lang/en.json [hunk 1]
```diff
@@ -172,6 +172,9 @@
     "GearType": "Type",
     "HangUp": "Hang-Up",
     "HangUpChoice": "Select Hang-Up",
+    "HealthBonus": "Bonus",
```
Comment (phildominguez) [inline lang/en.json:175]: Do we want to just make a single `"Bonus"` field? `"DefenseBonus"` is also just "Bonus".

File: lang/en.json [hunk 2]
```diff
@@ -172,6 +172,9 @@
...
+    "HealthCondition": "Conditioning",
+    "HealthOrigin": "Origin",
```
Comment (phildominguez) [inline lang/en.json:177]: I think we could just use `E20.SkillConditioning` and `E20.Origin` for these

File: module/documents/actor.mjs [hunk 3]
```diff
@@ -111,6 +112,27 @@ export class Essence20Actor extends Actor {
...
   }
 
+  /**
+  * Prepare Health specific data.
+  */
+  _prepareHealth () {
+    const system = this.system;
+    system.healthIsReadOnly = true;
+    const health = system.health;
+    let origin = 0;
```
Comment (phildominguez) [inline module/documents/actor.mjs:122]:
```suggestion
    let startingHealth = 0;
```

File: module/documents/actor.mjs [hunk 4]
```diff
@@ -111,6 +112,27 @@ export class Essence20Actor extends Actor {
...
+    const conditioning = system.conditioning;
+    const bonus = system.health.bonus;
+    const originName = game.i18n.localize('E20.HealthOrigin');
+    const conditionName = game.i18n.localize('E20.HealthCondition');
+    const bonusName = game.i18n.localize('E20.HealthBonus');
+    const origins = getItemsOfType ('origin', this.items);
+    if (origins.length>0) {
+      origin = origins[0].system.startingHealth;
+    }
```
Comment (phildominguez) [inline module/documents/actor.mjs:131]:
```suggestion
    const bonusName = game.i18n.localize('E20.HealthBonus');

    const origins = getItemsOfType('origin', this.items);
    if (origins.length) {
      origin = origins[0].system.startingHealth;
    }
```

**Review by phildominguez — CHANGES_REQUESTED** [review]
Seeing this when I delete my origin:
image

File: lang/en.json (hunk 1 above)
Comment (JFranklin3000) [reply lang/en.json:175]: Changed

File: lang/en.json (hunk 2 above)
Comment (JFranklin3000) [reply lang/en.json:177]: Changed

File: module/documents/actor.mjs (hunk 3 above)
Comment (JFranklin3000) [reply module/documents/actor.mjs:122]: Changed

**Review by phildominguez — APPROVED** [review]

**Pull request merged**